swc/private/versions.bzl

```diff
@@ -2,64 +2,14 @@
-        "linux-x64-gnu": "sha384-PwxqpU1ROBuiGtRdMGaupnNR2NjEUu+loz7rn9mOmf0wSuOmStRNm4SsStfCpgBq",
-        "win32-arm64-msvc": "sha384-NjRz1MgDghi5dxcDJFmnGX8oifL1y1UGKU0FE8O+x40Kri5qEJjEYGQxahGNEVMy",
-        "win32-ia32-msvc": "sha384-6sEP3EiORWnGTMcMPsIqj5x25JogO8nA5eQs5sU7C6xm+wWEYWLcr1kczphu5qgW",
-        "win32-x64-msvc": "sha384-ocyZh77sV7W1BTYHhH4sKbvdZg2wrJjl43I7HI1B5MLecGGZ6n7aOydoI/npmsVF",
-    },
-    "v1.3.35": {
-        "darwin-arm64": "sha384-sG4QGIP6wugBRMwXBT3b5AiSI2fHRPI1OMKLZQ8N6/4ztTthRZn5Tg0IBuJIcUyL",
-        "darwin-x64": "sha384-oFlWDyXjgxFBjq4HpESNQrIwd9SlIwFXvYLNa+Vm+KXODz6a2xidTJnm5qaZclDD",
-        "linux-arm-gnueabihf": "sha384-nx6UA7Xl4NUVfEXD64f0zXg/IPlcewGAy/tfLI+bR8eRIhxwYHnpARNutJt8JQ6y",
-        "linux-arm64-gnu": "sha384-EQxzix2cTg0vGHDqNT9i9OwBNojezsSFh6MMZaDpZm3A9b3ZK1zYJv+6JkWWD/hA",
-        "linux-x64-gnu": "sha384-kXYrgDDW6f7H3VrTJM5LRwGfk20XdqoLG6wDY73CbHvjMoqrWkwSgLSNostyx8oZ",
-        "win32-arm64-msvc": "sha384-1jP8HZ1qhHBuExAlEmP2A34RZFbViSCupmJk2nwNs2bPAER5eCzK74Jjnm+48+5K",
-        "win32-ia32-msvc": "sha384-pFaAn0A1ubc+Vafz/nwIyjazde7jfxMPZW9UGGB+D1J6NT1Ode3wr3p3+tacPSM+",
-        "win32-x64-msvc": "sha384-o1B3j63Zhvs7QJc/7iYuVwX+8QfiSm82AeT8/jW96K3F1kORNBF5b31KbnxsS3sT",
-    },
-    "v1.3.34": {
-        "darwin-arm64": "sha384-87HERB0qZ2UqjSxLMpQB7EtYASe7ovT36pPtj409p6eDtM9dAEEXQS10nJ/OOz3O",
-        "darwin-x64": "sha384-i3poY6tNdi83tkpaV+uDgTR1jYxHIXzeqS1KAi8+DZKHUvwlyEXmwDWly0kZPWDs",
-        "linux-arm-gnueabihf": "sha384-lJifb+KWZTa5xC7L9Q4KrI/UcgWWuaXvNp28z/phJmRz8kFIeDpxtWXc7WomUYYM",
-        "linux-arm64-gnu": "sha384-2Q5viIs4jkFd/H0abMrUhZP/ILfY3S3GceoU9vBfRfJgCj+GF9SARoCQa7cRYEBb",
-        "linux-x64-gnu": "sha384-k5POzu1Ub30FlqagD+Voq5vE/pwB6SB4WD2SH5Yg/ALdC7pligR0avdRyFjVZ2oS",
-        "win32-arm64-msvc": "sha384-JMwjte9/YE65JtnV5QXawhBTs4O6UNpFP47ZF5OB1cC6bMkonabh4YAjvJ3Pvze+",
-        "win32-ia32-msvc": "sha384-E6YvSV2WuqjTmphDnTrbCihlQmM5EIWqNd+LBbakGXOgQmPi3TXys3dohZwT+GCZ",
-        "win32-x64-msvc": "sha384-ZEfQyY08ElB6ktcnLFR7OB6pQ5QlMgdxhY/ENLCpcHOm8mktZsKN770/jM+lcJUX",
-    },
-    "v1.3.31": {
-        "darwin-arm64": "sha384-tSMDc6MG2llNW0PoEaWayHSia7YXDWVV+ceaiAl3Ew1xGA3lmfC2wTuKzaJU+16o",
-        "darwin-x64": "sha384-e9/dVt6KQTPVyH3y2Oy8J+Bz5XVI0hiK6fwknNsq/yTY+hftJ+SDqDW2YMkfCa3C",
-        "linux-arm-gnueabihf": "sha384-UCCjrMSg0dX5KN2IFCPhprsqLNoT7QXZwcKfpiYjhrpvIksCFq33nY8se+G88DKz",
-        "linux-arm64-gnu": "sha384-cipAIuV6Sa33feJMftOL56t3yEqr0/E6feFWkBnbIAtlSFsUEcv7LY+M7zLdn7Or",
-        "linux-x64-gnu": "sha384-s9hTnWi9jZM2MZJNnCn7Pg19vfSLnf8kCCYAgBN4UM4NFlN8+BpfGHG+jzq6BkaA",
-        "win32-arm64-msvc": "sha384-XYEEKojSlsPPTPi/BAxMM99DPtfaZ6oNkC+uyC55k4pYFIsHcKoGXN7CKhjfqKwJ",
-        "win32-ia32-msvc": "sha384-bEFBM1JCRTqtqx2jIVjF3PVyHRk84RWsAPR1asCVhcNMzvz1VDd6Ydtq2KR4K5v1",
-        "win32-x64-msvc": "sha384-wz5LmUtrIr3Qfy9yKbYWX+N2CqdmDyoe7OTnOkYaMR5s1v/e4s4BVrjFjQqkwcex",
-    },
-    "v1.3.25": {
```

Comment:
very few users are on windows, and removing all of these is a breaking change for them. Now that we are in an RC, we should avoid breaking changes.